  @Test
  public void invocation_leading_to_ArithmeticException() {
    CheckVerifier.newVerifier()
      .onFile("src/test/files/se/MethodInvocationLeadingToArithmeticException.java")
      .withCheck(new DivisionByZeroCheck())
      .withClassPath(SETestUtils.CLASS_PATH)
      .verifyIssues();
  }

